packages/react-native-win32/.gitignore

```diff
@@ -1,12 +1,12 @@
 /dist
 /flow
-/flow-typed
```

Comment:
>flow-typed [](start = 1, length = 10)

We haven't copied this in a while, since it's not part of the published RN package